packages/db/utils.js

```diff
@@ -106,3 +106,11 @@ const DEAL_STATUS = new Set([
   'published',
   'active'
 ])
+
+export function parseTextToNumber (numberText) {
+  const number = Number(numberText)
+  if (!Number.isSafeInteger(number)) {
+    throw Error('Invalid number')
```

Comment:
Nit.
I wonder if we could add a more specific Error.
I know it is a general util function and not tied to a specific piece of code, but suggesting a possible cause for this error could be the Postgres returning BigInts?
Just imagining a dev in the future looking at logs and struggling to figure this out?

Reply:
Yes, given that this is inside the `db` package, we can probably at least give a _suggestion_ that it might be BIGINT related.

Reply:
Added some documentation to make use explicit.

Reply:
Ta.